app/views/documents/index/_results.html.erb

```diff
@@ -38,6 +38,10 @@
           <%= t "documents.index.search_results.state",
             state: t("user_facing_states.#{document.user_facing_state}.name")
           %>
+          <br/>
+          <%= t "documents.index.search_results.last_edited_by",
+            user: document.last_editor&.name || document.creator&.name || I18n.t("documents.index.search_results.unknown_creator")
```

Comment:
Can we set last edited when we mark who created the document so that we don't need to do this or here? 

It feels like this should follow same pattern as db created_at and updated_at records where updated_at is set to same time as created_at when an item is created.

Reply:
Makes sense - added here: https://github.com/alphagov/content-publisher/pull/336/commits/6820219e0cc9ea8d06f07bb0d5627f132d360fac